src/data/core-dark.json

```diff
@@ -192,11 +192,11 @@
         "type": "color"
       },
       "600": {
-        "value": "#616568",
+        "value": "#98999B",
```

Comment:
Seems like all the system values went up by down, so this should be 500, etc. 100 is now #1A1D21

Reply:
@alexandra-lim Are you referencing this chart? Cause since we only have the spread from 100-800 for other colors, I actually omitted the #FFF from `grey.800`, since we have that as a white shade

<img width="275" alt="CleanShot 2023-08-23 at 09 41 31@2x" src="https://github.com/Fullscript/aviary-tokens/assets/5652496/bdc4b3fe-45f5-4f15-82fa-c3d40fe15d63">

Just edited the figma file to move to this, does this work/make sense or do you think its out of whack still
<img width="295" alt="CleanShot 2023-08-23 at 09 43 21@2x" src="https://github.com/Fullscript/aviary-tokens/assets/5652496/4d11eae3-4180-428a-9259-2a018b3942c9">